src/lib/icons/audio/RepeatOneIcon.svelte

```diff
@@ -1,16 +1,21 @@
 <script lang="ts">
     // From app-builders/source/scripture/app-android-scripture/src/main/res/drawable-xxxhdpi/ic_repeat_one_black_24.png
     // Converted with https://vectorizer.ai/
-    export let color = 'black';
+
+    interface Props {
```

Comment:
Remove `interface Props` since declaration with defaults is sufficient for the types.